### PR TITLE
Feature/jiahuichen

### PR DESCRIPTION
**Pull request by jchen23-stf**
# *Name of the PR*
Jiahui Chen assignment 1 

## :recycle: Current situation & Problem
*Describe the current situation (if possible, with an exemplary (or real) code snippet and/or where this is used)*
*Please link any open issue that is addressed with this PR*

## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ ] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: CS342Application.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -533,7 +540,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = edu.stanford.cs3422023application;
+				PRODUCT_BUNDLE_IDENTIFIER = edu.stanford.jchen23.cs3422023application;
```
Comment (vishnuravi) [inline CS342Application.xcodeproj/project.pbxproj]: Before merging your branch into the main application, I'd recommend going back in to Xcode and changing the bundle identifier back to `edu.stanford.cs3422023application`.

File: CS342ApplicationUITests/MemberViews/JiahuiChen.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import XCTest
+
+
+class JiahuiChen: XCTestCase {
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+    }
+    
+    func testRajPabariAndFunFactView() throws {
```
Comment (vishnuravi) [inline CS342ApplicationUITests/MemberViews/JiahuiChen.swift]:
```suggestion
    func testJiahuiChenAndFunFactView() throws {
```

File: CS342Application.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -518,7 +525,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 3833Q4CX9P;
```
Comment (vishnuravi) [inline CS342Application.xcodeproj/project.pbxproj]: Please make sure you reset the development team in Xcode also before you merge!

**Review by vishnuravi — APPROVED** [review]
Hey @jchen23-stf, this is really well done! Your view looks great and all the checks passed! 🚀

I'm approving this and you can go ahead and merge it, but please make sure to reset the bundle ID and development team (as noted below) so we don't run into issues when merging into the main branch with others' code!

File: CS342MemberViews/Sources/JiahuiChen/FunFact.swift [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct RickRoll: View {
```
Comment (vishnuravi) [inline CS342MemberViews/Sources/JiahuiChen/FunFact.swift:12]: Always love a good rick roll 😆
